Makefile

```diff
@@ -139,6 +139,10 @@ up-swarm-devel:
 	${DOCKER_COMPOSE} -f services/docker-compose.yml -f services/docker-compose.devel.yml -f services/docker-compose.tools.yml config > $(TEMPCOMPOSE).tmp-compose.yml
 	${DOCKER} stack deploy -c $(TEMPCOMPOSE).tmp-compose.yml services
 
+up-webclient-devel: up-swarm-devel remove-intermediate-file
```

Comment:
We plan to redo this calls (see #546)
In the meantime, please add ``up-webclient-devel`` in the ``.PHONY`` list (see line 127) and add help documentation above make target (see line 128 as example)

Reply:
Done